docker-compose.yaml

```diff
@@ -28,6 +28,12 @@ services:
             WORDPRESS_DB_USER: wordpress
             WORDPRESS_DB_PASSWORD: wordpress
             WORDPRESS_DB_NAME: wordpress
+            WORDPRESS_CONFIG_EXTRA: |
```

Comment:
I added these changes to make life easier. Thanks @jacobbednarz for this improvement.